pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LeastResourceUsageWithWeight.java

```diff
@@ -125,6 +125,11 @@ private double updateAndGetMaxResourceUsageWithWeight(String broker, BrokerData
     @Override
     public Optional<String> selectBroker(Set<String> candidates, BundleData bundleToAssign, LoadData loadData,
                                          ServiceConfiguration conf) {
+        if (candidates.isEmpty()) {
+            log.error("There are no available brokers as candidates at this point for bundle: {}", bundleToAssign);
```

Comment:
I won't use "error", but "info". Error is scary

Reply:
Ok, changed` "error" to "info".  I find if not select any broker  the `getLeastLoadedFromLoadManager` has a warn , it's enough for us. PTAl.